src/Util/Log/JUnit.php

```diff
@@ -229,6 +229,7 @@ public function startTestSuite(TestSuite $suite)
     {
         $testSuite = $this->document->createElement('testsuite');
         $testSuite->setAttribute('name', $suite->getName());
+        $testSuite->setAttribute('package', $suite->getName());
```

Comment:
My guess is that JUnit uses the name of the package the class belongs to. In PHP that would be the namespace of the class (or `Vendor_Package` in case the class is not namespaced and its name is `Vendor_Package_Class`).